roles/openshift_loadbalancer/tasks/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
+- fail: msg="Cannot use containerized=true for load balancer hosts."
```

Comment:
Perhaps suggest adding containerized=false to [lb] in the example hosts file or hosts within it with a note about why its necessary?